code/modules/antagonists/changeling/powers/augmented_eyesight.dm

```diff
@@ -19,7 +19,7 @@
 	else
 		to_chat(user, "We can't adjust our eyes if we don't have any!")
 
-/datum/action/changeling/augmented_eyesight/sting_action(mob/living/carbon/human/user)
+/datum/action/changeling/augmented_eyesight/sting_action(mob/living/carbon/user)
```

Comment:
This shouldn't change anything.
You can call this function with a monkey as the type, and it will work as long as you don't use any vars that are specific to human only (which is true, since otherwise this wouldn't compile)

Reply:
But it does, without the change you cannot toggle it as a monkey, with the change you can

Reply:
Oh I see, there is a typecheck in the proc